src/main/scala/org/hammerlab/guacamole/commands/SomaticStandardCaller.scala

```diff
@@ -113,6 +117,28 @@ object SomaticStandard {
           args.minTumorAlternateReadDepth
         )
 
+      if (args.dbSnpVcf != "") {
+        val adamContext = new ADAMContext(sc)
+        val dbSnpVariants = adamContext.loadVariantAnnotations(args.dbSnpVcf)
+        potentialGenotypes
+          .keyBy(_.adamVariant)
+          .leftOuterJoin(dbSnpVariants.keyBy(_.variant))
```

Comment:
`s/_.variant/_.getVariant/`? IJ telling me that the direct accessor is deprecated